tt_metal/impl/device/program_cache.hpp

```diff
@@ -51,7 +51,7 @@ struct ProgramCache {
 
    private:
     inline static bool is_enabled_ = false;
-    std::unordered_map<uint64_t, tt::stl::unique_any<1024, 32>> cache_{};
+    std::unordered_map<uint64_t, tt::stl::unique_any<1152, 32>> cache_{};
```

Comment:
why?

Reply:
@ayerofieiev-tt let me double check if this was necessary. I might remove it